src/main/java/graphql/language/ArrayValue.java

```diff
@@ -87,7 +91,8 @@ public String toString() {
 
     @Override
     public ArrayValue deepCopy() {
-        return new ArrayValue(deepCopy(values), getSourceLocation(), getComments(), getIgnoredChars(), getAdditionalData());
+        List<Value> copiedValues = assertNotNull(deepCopy(values));
```

Comment:
Add an assert here to keep NullAway happy

In practice it is not possible to have null values. Have to put this here otherwise you'll get a NullAway error